src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -26,6 +27,8 @@ public sealed class SparkContext : IJvmObjectReferenceProvider
 
         private readonly SparkConf _conf;
 
+        private readonly string _version;
```

Comment:
let's not cache this and do `public string Version() => (string)_jvmObject.Invoke("version")` for simplicity.

Reply:
simplified, thanks for your paitence!